src/SUnit-Core/TestResult.class.st

```diff
@@ -263,6 +274,12 @@ TestResult >> failureCount [
 
 { #category : #compatibility }
 TestResult >> failures [
+	"Don't half bind me. I return failures tests.
```

Comment:
What does "don't half bind me" here? Could the comment be a little more explicit?